From b75f9e83976d9b6737943f9658253a4c67e665ba Mon Sep 17 00:00:00 2001
From: Gustav Grusell <gustav.grusell@gmail.com>
Date: Thu, 18 May 2023 20:26:58 +0200
Subject: [PATCH] build: add meson build file

Signed-off-by: Gustav Grusell <gustav.grusell@gmail.com>
---
 meson.build | 12 ++++++++++++
 1 file changed, 12 insertions(+)
 create mode 100644 meson.build

diff --git a/meson.build b/meson.build
new file mode 100644
index 0000000..4f2c4d8
--- /dev/null
+++ b/meson.build
@@ -0,0 +1,12 @@
+project('platformfolders', 'cpp', default_options : ['cpp_std=c++11', 'warning_level=3'])
+
+platformfolders_inc = include_directories('sago')
+
+
+platformfolders_lib = static_library('platform_folders', 'sago/platform_folders.cpp', include_directories: platformfolders_inc)
+
+
+platformfolders_dep = declare_dependency(
+    include_directories : platformfolders_inc,
+    link_with : platformfolders_lib
+)
-- 
2.37.2

